scala/scala_import.bzl

```diff
@@ -1,5 +1,40 @@
 load("@io_bazel_rules_scala//scala:jars_to_labels.bzl", "JarsToLabelsInfo")
 
+def _stamp_symlinked_jar(ctx, jar):
+    symlink_file = ctx.actions.declare_file(jar.basename)
+    ctx.actions.symlink(output = symlink_file, target_file = jar)
+
+    stamped_jar_filename = jar.basename.rstrip(".jar") + "-stamped.jar"
+
+    # Preferred way, but currently broken:
+    # java toolchain's ijar incorrectly handles MANIFEST sections
+    # https://github.com/bazelbuild/bazel/issues/12730
+    #    return java_common.stamp_jar(
+    #        actions = ctx.actions,
+    #        jar = symlink_file,
+    #        target_label = ctx.label,
+    #        java_toolchain = ctx.attr._java_toolchain[java_common.JavaToolchainInfo],
+    #    )
+
+    # Stamp with custom built (https://github.com/bazelbuild/bazel/pull/12771)
+    stamped_file = ctx.actions.declare_file(stamped_jar_filename)
+
+    ctx.actions.run(
+        executable = ctx.executable._ijar,
+        inputs = [symlink_file],
+        outputs = [stamped_file],
+        arguments = [
+            "--nostrip_jar",
+            "--target_label",
+            ctx.label.name,
+            symlink_file.path,
```

Comment:
Why symlink is needed? Can't you just pass `jar.path`?

Reply:
Because bazel does not allow modifications outside package